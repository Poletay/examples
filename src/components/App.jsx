import React from 'react';

const App = () => <div className="container">Реакт в деле!!!</div>;

export default App;
