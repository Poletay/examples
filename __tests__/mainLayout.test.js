import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/components/App';

jest.mock('../src/components/Header', () => 'Header');
jest.mock('../src/components/Footer', () => 'Footer');
jest.mock('../src/components/LeftContent', () => 'LeftContent');
jest.mock('../src/components/MiddleContent', () => 'MiddleContent');
jest.mock('../src/components/RightContent', () => 'RightContent');

test('example', () => {
  const component = renderer.create(
    <App />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
