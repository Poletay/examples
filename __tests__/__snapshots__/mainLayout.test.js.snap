// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example 1`] = `
<div
  className="main"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <Header />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-3"
    >
      <LeftContent />
    </div>
    <div
      className="col-7"
    >
      <MiddleContent />
    </div>
    <div
      className="col-2"
    >
      <RightContent />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <Footer />
    </div>
  </div>
</div>
`;
